lib/Common/Core/SysInfo.cpp

```diff
@@ -254,7 +254,7 @@ AutoSystemInfo::SSE2Available() const
     return false; // TODO: xplat support
 #endif
 #else
-    #error Unsupported platform.
+    return false;
```

Comment:
I'd leave the error here and add an explicit ifdef for ARMv7/Android here so that anyone adding new platform support is aware that this function is something they need to think about